clu/src/lib.rs

```diff
@@ -205,13 +205,13 @@ fn linux_init(
     let mesh_ip = SETTINGS.read().unwrap().network.own_ip.clone();
     let our_exit_ip = SETTINGS.read().unwrap().exit_client.exit_ip.clone();
 
-    request_own_exit_ip(SETTINGS.clone());
+    request_own_exit_ip(SETTINGS.clone())?;
```

Comment:
think we should just delete line 208 and 209, request own exit ip returns error when the exit is unreachable, which it is on init (the loop further down handles retrying that). Early returning from this function breaks rita